topology.go

```diff
@@ -45,6 +45,8 @@ func (r *Router) Topology() TopologyController {
 }
 
 func (r *Router) AddInstance(ctx context.Context, rsID uuid.UUID, info InstanceInfo) error {
+	r.log().Debugf(ctx, "trying to add instance %s to router topology in rs %s", info, rsID)
```

Comment:
we can remove String() method for InstanceInfo and log using `%s` -> `%+v`:

```golang
r.log().Debugf(ctx, "trying to add instance %+v to router topology in rs %s", info, rsID)
```

Reply:
this may lead to the fact that if someone adds some new fields, we will be less stable in the logging interface. The idea is to limit the dependency of changes